content/news/buzz/august-2023-status-update/contents.lr

```diff
@@ -0,0 +1,53 @@
+
+* We added a `Texual backend for Toga <https://github.com/beeware/toga/pull/2065>`__. This means the same Toga application will now run on 3 desktop platforms, 2 mobile platforms, as a single page web app, and in a terminal window. As with the web backend, there's still a lot of work to fill out all the widgets, but we've got enough in place to run the BeeWare tutorial app and the first 2 Toga tutorial steps.
+
+* We `improved the architecture and word size checks that Briefcase performs when building apps <https://github.com/beeware/briefcase/pull/1360>`__. This effectively adds support for 32-bit x86 and ARM platforms; we're limited in our ability to perform automatic tests to verify these platforms work as advertised, and upstream tooling isn't always up to the task (for example, Linuxdeploy doesn't support ARM at any word size) - but Briefcase now performs much more robust checks, and will make a best effort when support *should* exist.
+
+* We added `support for openSUSE (and other SUSE-derived distributions) <https://github.com/beeware/briefcase/pull/1417>`__ to Briefcase.
+
+* We `updated the version of the Android command line tools that Briefcase uses <https://github.com/beeware/briefcase/pull/1397>`__.
+
+* We resolved a long standing issue with the use of `Objective C blocks on M1 hardware <https://github.com/beeware/rubicon-objc/pull/357>`__.
+
+What's next?
+--------------
+
+Based on current progress, it seems highly likely that the Toga audit will be complete on Windows and Android by the end of September. This will bring the Toga audit to completion, providing a very stable foundation for future work on Toga. It also means you can expect a Toga v0.3.2 release in the coming month, incorporating the myriad bug fixes and improvements that the audit has revealed. This release will include a number of backwards-incompatible changes identified as part of the audit, so be prepared for a bumpier-than-usual upgrade process.
+
+So - what comes after the audit? Firstly, we'll be looking into iOS App Store submission - we've been made aware that although Briefcase apps work fine on both device and simulator, the App Store has been rejecting apps because of an internal layout issue. We have already begun work on this, and thanks to a contribution from the community, we have a very strong foundation for this work. We'll also start looking at Python 3.12 support, and being preparations for upstreaming our iOS patches into CPython core.
```

Comment:
```suggestion
So - what comes after the audit? Firstly, we'll be looking into iOS App Store submission - we've been made aware that although Briefcase apps work fine on both device and simulator, the App Store has been rejecting apps because of an internal layout issue. We have already begun work on this, and thanks to a contribution from the community, we have a very strong foundation for this work. We'll also start looking at Python 3.12 support, and making preparations for upstreaming our iOS patches into CPython core.
```